src/AssemblySharedInfoGenerator/AssemblySharedInfo.cs

```diff
@@ -45,7 +45,7 @@
 // to distinguish one build from another. AssemblyFileVersion is specified
 // in AssemblyVersionInfo.cs so that it can be easily incremented by the
 // automated build process.
-[assembly: AssemblyVersion("1.0.0.774")]
+[assembly: AssemblyVersion("1.0.0.812")]
```

Comment:
Please exclude `AssemblySharedInfo.cs` always with your PR, it is auto-generated during build anyway.